tests/upgrade/test_migrate.py

```diff
@@ -34,16 +54,32 @@ def new_strategy(vault_proxy, proxyAdmin, deployer):
 
     ## We delete from deploy and then fetch again so we can interact
     AdminUpgradeabilityProxy.remove(strat_proxy)
-    strat_proxy = AuraBalStakerStrategy.at(strat_proxy.address)
+    return AuraBalStakerStrategy.at(strat_proxy.address, owner=gov)
+
+
+@pytest.fixture
+def new_bb_a_usd(new_strategy):
+    return interface.ERC20(new_strategy.BB_A_USD())
 
-    return strat_proxy
 
 
-def test_check_storage_integrity(strat_proxy, vault_proxy, new_strategy):
-    with brownie.reverts():
-        strat_proxy.minBbaUsdHarvest()
+def test_migrate(strat_proxy, vault_proxy, new_strategy, current_bb_a_usd, new_bb_a_usd, gov, want):
+    ## 1. Reduce "minBbaUsdHarvest" threshold on old strategy and Harvest
+    current_bbausd_bal = current_bb_a_usd.balanceOf(strat_proxy)
+    strat_proxy.setMinBbaUsdHarvest(current_bbausd_bal - 1)
```

Comment:
I think we should just set this to 0 in the actual migration script

Reply:
Sounds good to me